data-prepper-plugins/opensearch/src/integrationTest/java/org/opensearch/dataprepper/plugins/sink/opensearch/OpenSearchSinkIT.java

```diff
@@ -532,6 +532,54 @@ public void testEventOutput() throws IOException, InterruptedException {
     sink.shutdown();
   }
 
+  @Test
+  public void testOpenSearchDocumentId() throws IOException, InterruptedException {
+    final String expectedId = "id1";
+    final String testIndexAlias = "test_index";
+    final Event testEvent = JacksonEvent.builder()
+            .withData("{\"info\": {\"ids\": {\"id\": \""+expectedId+"\"}, \"val\":\"200\"}, \"name\":\"id_test\" }")
```

Comment:
Using a `Map` would probably make this a little more readable. If not that, make `String.format` will help with readability.